src/libs/ReportActionsUtils.js

```diff
@@ -580,6 +595,93 @@ function isMessageDeleted(reportAction) {
     return lodashGet(reportAction, ['message', 0, 'isDeletedParentAction'], false);
 }
 
+/**
+ * Get the transactions related to a report preview
+ *
+ * @param {Object} reportPreviewAction
+ * @returns {Object}
+ */
+function getReportPreviewTransactionsWithReceipts(reportPreviewAction) {
+    const transactionIDs = lodashGet(reportPreviewAction, ['childLastReceiptTransactionIDs'], '').split(',');
+    return _.reduce(
+        transactionIDs,
+        (transactions, transactionID) => {
+            const transaction = allTransactions[transactionID];
+            if (transaction) {
+                transactions.push(transaction);
+            }
+            return transactions;
+        },
+        [],
+    );
+}
+
+/**
+ * @param {Object} iouReportAction
+ * @returns {Object}
+ */
+function getTransaction(iouReportAction) {
+    const transactionID = lodashGet(iouReportAction, ['originalMessage', 'IOUTransactionID']);
+    return allTransactions[transactionID] || {};
+}
+
```

Comment:
We already have this method https://github.com/Expensify/App/blob/61ab4bcf100500ce49bf1e2dfaa128f9511ea941/src/libs/TransactionUtils.js#L94-L102 we should reuse it

Reply:
Ah, this didn't get caught by the merge.